pkg/scheduler/internal/cache/snapshot_test.go

```diff
@@ -229,3 +230,107 @@ func TestCreateUsedPVCSet(t *testing.T) {
 		})
 	}
 }
+
+func TestNewSnapshot(t *testing.T) {
+	testCases := []struct {
+		name                         string
+		pods                         []*v1.Pod
+		nodes                        []*v1.Node
+		expectedNumNodes             int
+		expectedPodsWithAffinity     int
+		expectedPodsWithAntiAffinity int
```

Comment:
Add a new field `expectedUsedPVCSet`, and adding test cases like `multiple nodes, pods with PVCs`.